code/obj/item/mob_parts.dm

```diff
@@ -259,66 +259,35 @@ ABSTRACT_TYPE(/obj/item/parts)
 		return object
 
 	//for humans
-	attach(var/mob/living/carbon/human/attachee,var/mob/attacher,var/both_legs = 0)
-		if(!can_act(attacher))
-			return
-		if(!src.easy_attach)
-			if(!surgeryCheck(attachee, attacher))
+	attach(var/mob/living/carbon/human/attachee,var/mob/attacher)
+		if(!ishuman(attachee) || attachee.limbs.vars[src.slot])
+			return ..()
+		if(attacher)
+			if(!can_act(attacher))
 				return
-
-		if(!both_legs)
-			if(attacher.zone_sel.selecting != slot || !ishuman(attachee))
+			if(!src.easy_attach)
+				if(!surgeryCheck(attachee, attacher))
+					return
+			if(attacher.zone_sel.selecting != slot)
 				return ..()
 
-			if(attachee.limbs.vars[src.slot])
-				boutput(attacher, "<span class='alert'>[attachee.name] already has one of those!</span>")
-				return
-
-			attachee.limbs.vars[src.slot] = src
-		else
-			if (!(attacher.zone_sel.selecting in list("l_leg","r_leg")))
-				return ..()
-			else if(attachee.limbs.vars["l_leg"] || attachee.limbs.vars["r_leg"])
-				boutput(attacher, "<span class='alert'>[attachee.name] still has one leg!</span>")
-				return
+			attacher.remove_item(src)
 
-			attachee.limbs.l_leg = src
-			attachee.limbs.r_leg = src
+			playsound(attachee, "sound/effects/attach.ogg", 50, 1)
```

Comment:
single quotes are proffered when referencing sound files that don't require string operations

Reply:
I always get confused about which to use, thanks!